Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/MotionController/AttachToController.cs

```diff
@@ -1,47 +1,37 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using Microsoft.MixedReality.Toolkit.Internal.Attributes;
 using Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices;
 using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
 using Microsoft.MixedReality.Toolkit.Internal.EventDatum.Input;
 using Microsoft.MixedReality.Toolkit.Internal.Extensions;
-using Microsoft.MixedReality.Toolkit.Internal.Interfaces;
 using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem.Handlers;
 using Microsoft.MixedReality.Toolkit.SDK.Input;
 using UnityEngine;
 
 namespace Microsoft.MixedReality.Toolkit.SDK.UX.MotionController
 {
     /// <summary>
-    /// Waits for a controller to be initialized, then attaches itself to a specified element
+    /// Waits for a controller to be initialized, then attaches itself to a specified hand.
     /// </summary>
     public class AttachToController : InputSystemGlobalListener, IMixedRealitySourcePoseHandler
     {
         [SerializeField]
-        protected Vector3 PositionOffset = Vector3.zero;
+        private Handedness handedness = Handedness.Left;
```

Comment:
do we want to initialize to Handedness.None?

Reply:
Pretty sure the default (even in master) is left. What's the point of attaching to a controller of it doesn't have a hand to attach to?

Reply:
Pretty sure the default (even in master) is left. What's the point of attaching to a controller of it doesn't have a hand to attach to?

Reply:
Default in master is left; default in June is unknown.

https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/june18_dev/Assets/HoloToolkit/Input/Scripts/Utilities/ControllerFinder.cs#L42

This was changed to facilitate spawning AttachToController/ControllerFinders at runtime, so it doesn't try to attach to a controller in OnEnable (which happens on Instantiate) and then get its handedness changed in Start (which happens after settings can be changed on the script).